Tests/Marketplace/install_packs.sh

```diff
@@ -20,36 +27,37 @@ fi
 
 echo "Trying to authenticate with GCS..."
 gcloud auth activate-service-account --key-file="$GCS_MARKET_KEY" > auth.out 2>&1
-if [ $? -ne 0 ]; then
-  echo "Failed to authenticate with GCS, exiting..."
-  exit 1
-fi
+exit_on_error $? "Failed to authenticate with GCS, exiting..."
+
 echo "Successfully authenticated with GCS."
 
 echo "starting configure_and_install_packs instance role: ${INSTANCE_ROLE}"
-exit_code=0
-if [[ "${INSTANCE_ROLE}" == "XSIAM" ]]; then
 
+if [[ "${INSTANCE_ROLE}" == "XSIAM" ]]; then
   if [ -n "${CLOUD_CHOSEN_MACHINE_IDS}" ]; then
+    exit_code=0
```

Comment:
Add a log message that we exited in a good way, because it's not a cloud machine, and note the machine (to eliminate "why didn't you run on 'my' machine?".)